### PR TITLE
DPL: insert timeout to avoid OOM killer when runningh with DDS

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

File: Framework/Core/src/DDSConfigHelpers.cxx [hunk 1]
```diff
@@ -117,6 +116,8 @@ void dumpDeviceSpec2DDS(std::ostream& out,
     }
   }
 
+  float timeout = 0.0;
```
Comment (ktf) [inline Framework/Core/src/DDSConfigHelpers.cxx:119]: @davidrohr this starts from 0, in the chat you had 0.2, was that intended?

**Review by davidrohr — APPROVED** [review]
thx, trying on EPN

File: Framework/Core/src/DDSConfigHelpers.cxx (hunk 1 above)
Comment (davidrohr) [reply Framework/Core/src/DDSConfigHelpers.cxx:119]: no, from 0 is fine of course :)

**Review by davidrohr — APPROVED** [review]
perfect, works, shall go into tomorrow's nightly

**Pull request merged**